Asgardia.Constitution.Proposal.txt

```diff
@@ -310,7 +310,7 @@ CHAPTER 8. ADMINISTRATION OF GOVERNMENT
 
 Article 31. Legal System and Branch structures
 § 1. The Constitution of Asgardia is the foundation of Asgardia’s legal system. Asgardia’s Constitution takes precedence over all other legal documents and is directly applicable.
-§ 2. Asgardia’s legal acts include:
+§ 2. Asgardia’s legal acts include administering:

```

Comment:
I was not sure if "administering" is the correct verb here for everything. If each of the sub-bullets requires a different verb, we'll need to add one for each one because it doesn't make sense without verbs.